styles/global.css

```diff
@@ -76,20 +120,73 @@ body {
 }
 
 .dropdown-content {
-  background-color: rgb(253, 252, 252);
+  background-color: black;
+  /* hide dropdown content by default */
   display: none;
   width: 100px;
+  /* make content appear below dropdown button */
   position: absolute;
+  top: 80%;
```

Comment:
Can you try to align so that the dropdown start precisely at the bottom of the nav item
![CleanShot 2024-10-18 at 15 29 09](https://github.com/user-attachments/assets/f0b27176-1cbb-4af3-92b9-b4072b50411c)

Reply:
Yes, I changed the top of dropdown-content to be 100% to place it right below the dropdown button.

Reply:
Much better. Still looks a bit strange. Can you try making the dropdown to be equal width to its parents
![Uploading CleanShot 2024-10-21 at 11.50.17@2x.png…]()

Reply:
yes, I added the property box-sizing:border-box to included all padding and border to the width so that the property width:100% can work better